torch_xla/csrc/ops/amax.cpp

```diff
@@ -0,0 +1,49 @@
+#include "torch_xla/csrc/ops/amax.h"
+
+#include "tensorflow/compiler/xla/xla_client/util.h"
+#include "torch_xla/csrc/lowering_context.h"
+#include "torch_xla/csrc/ops/infer_output_shape.h"
+#include "torch_xla/csrc/reduction.h"
+
+namespace torch_xla {
+namespace ir {
+namespace ops {
+namespace {
+
+xla::Shape NodeOutputShape(const Value& input,
+                           std::vector<xla::int64>& dimensions, bool keepdim) {
+  auto lower_for_shape_fn =
+      [&](absl::Span<const xla::XlaOp> operands) -> xla::XlaOp {
+    return BuildMaxInDims(operands[0], dimensions, keepdim);
+  };
+  return InferOutputShape({input.shape()}, lower_for_shape_fn);
+}
+
+}  // namespace
+
+AMax::AMax(const Value& input, std::vector<xla::int64> dimensions, bool keepdim)
```

Comment:
Maybe change to `Amax` instead of `AMax`? I think we also have `Acos` and `Asin`. It is better to keep the same style.

Reply:
Addressed.